### PR TITLE
Gds 446 link list

### PR DESCRIPTION
**Pull request by Lesz**
Hi Mason and Sean, 

this is my first go at the link list - please have a look and let me know your feedback. I'm pretty sure that I probably could have done things different to better fit your framework. So I'm more than happy to change/modify my CSS/HTML :-)

Cheers

### CODE REVIEW

**Comment by seanelliott86** [comment]
@Lesz can you repoint your PR to develop please buddy?

File: src/components/link-list/_link-list.hbs [hunk 1]
```diff
@@ -1,3 +1,23 @@
-<div class="nsw-content nsw-container">
-	<h1>Link list</h1>      
-</div>
\ No newline at end of file
+<div class="nsw-container">
+	<h1>Link list</h1>  
+    <h2 style="margin-bottom:3rem;">Lesz testing</h2> 
+  
+  
+<nav aria-label="{{label}}" class="nsw-link-list">
```
Comment (masonphanNSW) [inline src/components/link-list/_link-list.hbs]: Can we indent everything inside the container div.

Comment (seanelliott86) [inline src/components/link-list/_link-list.hbs]: It is not essential to have all links in a `nav` element. `nav` is intended only for major block of navigation links that are consistent across the site. I would suggest changing from `nav` to a `div` and removing the `aria-label`, if this needs to change later we can review it again.

File: src/components/link-list/_link-list.scss [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    @include nsw-hover;
+    }
+
+  &__link:focus{
+    @include nsw-focus;
+    }
+
+  &__icon {
+    @include nsw-spacing(margin-left, md);
+    width: 24px;
```
Comment (seanelliott86) [inline src/components/link-list/_link-list.scss]: Use the `rem()` function here for your width and height px values `width: rem(24px)` we want to have everything in relative units.

**Pull request merged**